openstack/nova-hypervisor-agents/templates/_helpers.tpl

```diff
@@ -10,10 +10,25 @@ https://identity-3.{{.Values.global.region}}.{{.Values.global.tld}}/v3
 enabled = {{ $config.enabled }}
 {{- if $config.enabled }}
 novncproxy_base_url = https://{{include "nova_console_endpoint_host_public" .}}:{{ .Values.global.novaConsolePortPublic }}/{{ $cell_name }}/novnc/vnc_auto.html?path=/{{ $cell_name }}/novnc/websockify
+server_listen = $my_ip
+server_proxyclient_address = $my_ip
```

Comment:
This seems unrelated to serialproxy

Reply:
nope, it's crutial: https://docs.openstack.org/nova/latest/configuration/config.html#serial_console.proxyclient_address

This is the IP that the compute agent provides to the serialproxy service to connect to the Serial Port TCP port.